src/glean.js

```diff
@@ -8,21 +8,25 @@ const Storage = require('./storage');
 
 
 class Glean {
-  constructor(appId) {
-    this._eventStorage = new Storage(appId)
+  constructor() {
+    console.log("Initialize");
+    let gleanScript = document.querySelector('[src*=glean\\.js]');
+    let appId = gleanScript && gleanScript.getAttribute('app-id');
```

Comment:
Should we stop and not initialize further in case app-id is not present?

Reply:
Good point